examples/Makefile.am

```diff
@@ -70,15 +63,42 @@ endif # BUILD_EXAMPLES
 if BUILD_EXAMPLES_SOURCE
 EXAMPLES_DIR = $(DESTDIR)$(datadir)/libcoap/examples
 EXAMPLES_SRC = coap-client.c coap-server.c
+endif # BUILD_EXAMPLES_SOURCE
 
 ## Install example files
 install-exec-hook:
+if BUILD_EXAMPLES_SOURCE
 	$(MKDIR_P) $(EXAMPLES_DIR)
 	(cd $(top_srcdir)/examples ; \
 	$(INSTALL_DATA) $(EXAMPLES_SRC) ../LICENSE ../COPYING $(EXAMPLES_DIR) ; \
 	$(INSTALL_DATA) share.libcoap.examples.Makefile $(EXAMPLES_DIR)/Makefile; \
 	$(INSTALL_DATA) share.libcoap.examples.README $(EXAMPLES_DIR)/README)
+endif # BUILD_EXAMPLES_SOURCE
+if BUILD_ADD_DEFAULT_NAMES
+	if [ -d "$(DESTDIR)$(bindir)" ] ; then \
+		(cd $(DESTDIR)$(bindir) ; \
```

Comment:
This needs to be s/;/&&/ to avoid unwanted side effects in the current directory in case `cd $(DESTDIR)$(bindir)` fails (usually because of missing access permissions).

Reply:
Updated (and parenthesized the following set of if statements).  Changes pushed.

Reply:
Excellent, thanks!